### DISCUSSION TITLE
Feature: Sort "Key" and "Mapping" columns

### DISCUSSION THREAD

**Discussion by KayTaneJupiter** [sezanzeb/input-remapper#163] [Ideas]
Ability to sort the "Key" and "Mapping" columns of the GUI would be useful. I am often hunting through the list looking for the key I want to change.

Also, a general "Comment" column would also be useful to note what each key mapping does.

Thanks for the work on this project.

**Comment by sezanzeb** [comment]
Great ideas! I have added the comment feature to the macro-editor roadmap: https://github.com/sezanzeb/key-mapper/issues/109
